src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkImageBrowser.tsx

```diff
@@ -108,6 +117,15 @@ export class SmallArtworkImageBrowser extends React.Component<
   ArtworkBrowserProps,
   ArtworkBrowserState
 > {
+  slider: Slider
+
+  setSliderRef = slider => {
+    this.slider = slider
+    if (this.props.sliderRef) {
+      this.props.sliderRef(slider)
+    }
+  }
```

Comment:
It turns out that `SmallArtworkImageBrowser` doesn't need to maintain a reference to the slider so we can delete these two things (`slider` and `setSliderRef`) and just pass `this.props.sliderRef` directly to the `ref` property on the `Slider` element.